app/controllers/application_controller/compare.rb

```diff
@@ -1818,6 +1818,14 @@ def comp_record_data_compressed_nonexistsmode(idx, match, val, basval)
     row
   end
 
+  def comp_record_data_compressed_nonexistsmode(idx, match, val, basval)
+    comp_record_data_nonexistsmode(idx, match, val, basval)
+  end
+
+  def comp_record_data_expanded_nonexistsmode(idx, match, val, basval)
+    comp_record_data_nonexistsmode(idx, match, val, basval)
+  end
```

Comment:
@chrisarcand I want to unwind the de-duping in multiple PRs to make the reviews easier.